src/shared/pages/team/createTeam/initialState.ts

```diff
@@ -0,0 +1,17 @@
+import { State } from './state';
+
```

Comment:
The team data could be stored directly here, no need to nest it in a team object.

Reply:
Discussed with @parth-ho and agreed that this could be considered a valid approach.